test/pod.persona.test.ts

```diff
@@ -0,0 +1,229 @@
+  const personas = await pod.getPersonas('0x85760ef61c0ccB7BCC4C7A0116d80D59D92e736d');
+
+  await pod.callAsPersona(
+    pod.mintMember,
+    [userAddress2],
+    personas[0],
+    '0x85760ef61c0ccB7BCC4C7A0116d80D59D92e736d',
+  );
+  expect(create).toHaveBeenCalled();
+});
+
+test('Mint via adminPodMember persona should throw if sender is not an admin pod member', async () => {
+  jest
+    .spyOn(axios, 'post')
+    .mockResolvedValueOnce(constructGqlGetUsers(['0xf0C7d25c942264D6F21871c05d3dB3b98344b499']));
+
+  const pod = await getPod('1-root-as-admin.pod.eth');
+  pod.getPersonas = jest.fn().mockResolvedValue([
+    {
+      type: 'adminPodMember',
+      address: '0xBe71ECaA104645ab78ed62A52763b2854e6DaD2E',
+    },
+  ]);
+  const personas = await pod.getPersonas('0x08A9C4898e045A4E176C2cbf4Ca86aEf9f6EA478');
+
+  await expect(
+    pod.callAsPersona(pod.mintMember, [userAddress2], personas[0], userAddress2),
+  ).rejects.toThrow('Sender must be a member of the admin pod');
+});
+
+test('Mint via subPodMember persona creates a proposal', async () => {
+  mockGetPodFetchersByAddress();
+  jest
+    .spyOn(axios, 'post')
+    .mockResolvedValue(constructGqlGetUsers(['0x85760ef61c0ccB7BCC4C7A0116d80D59D92e736d']));
+  const create = jest
+    .spyOn(createSafe, 'createSafeTransaction')
+    .mockResolvedValue(getSafeTransactionFixture());
+
+  const pod = await getPod('1-member-pods.pod.eth');
+  const personas = await pod.getPersonas('0x85760ef61c0ccB7BCC4C7A0116d80D59D92e736d');
+
+  await pod.callAsPersona(
+    pod.mintMember,
+    [userAddress2],
+    personas[0],
+    '0x85760ef61c0ccB7BCC4C7A0116d80D59D92e736d',
+  );
+  expect(create).toHaveBeenCalled();
+});
+
+test('Mint via subPodMember persona should throw if sender is not a sub pod member', async () => {
+  jest
+    .spyOn(axios, 'post')
+    .mockResolvedValueOnce(constructGqlGetUsers(['0xf0C7d25c942264D6F21871c05d3dB3b98344b499']));
+
+  const pod = await getPod('1-root-as-admin.pod.eth');
+  pod.getPersonas = jest.fn().mockResolvedValue([
+    {
+      type: 'subPodMember',
+      address: '0x203D96211d5f0e0De9788302505dd37D26a28005',
+    },
+  ]);
+  const personas = await pod.getPersonas('0x08A9C4898e045A4E176C2cbf4Ca86aEf9f6EA478');
+
+  await expect(
+    pod.callAsPersona(pod.mintMember, [userAddress2], personas[0], userAddress2),
+  ).rejects.toThrow('Sub pod was not a member of this pod');
+});
+
+test('Mint via subPodMember persona should throw if the sub pod is not a member of the pod', async () => {
+  const pod = await getPod(expectedPod.name);
+  pod.getPersonas = jest.fn().mockResolvedValue([
+    {
+      type: 'subPodMember',
+      address: '0x203D96211d5f0e0De9788302505dd37D26a28005',
+    },
+  ]);
+  const personas = await pod.getPersonas('0x08A9C4898e045A4E176C2cbf4Ca86aEf9f6EA478');
+
+  await expect(pod.callAsPersona(pod.mintMember, [userAddress2], personas[0])).rejects.toThrow(
+    'Sub pod was not a member of this pod',
+  );
+});
```

Comment:
🤔 Do you mind breaking these tests up by each persona type?
Ex: move line 51-104 into a describe block, like 
```
describe('admin member', () => {
...
})
```